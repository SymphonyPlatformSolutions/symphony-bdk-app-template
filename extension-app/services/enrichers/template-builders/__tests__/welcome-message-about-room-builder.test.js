import WelcomeMessageAboutRoomBuilder from '../welcome-message-about-room-builder';

const data = {
  room_name: 'All Technicians',
};

describe('Welcome Message About Room Builder', () => {
  it('Should build message', () => {
    expect(WelcomeMessageAboutRoomBuilder.build(data)).toMatchSnapshot();
  });
});
