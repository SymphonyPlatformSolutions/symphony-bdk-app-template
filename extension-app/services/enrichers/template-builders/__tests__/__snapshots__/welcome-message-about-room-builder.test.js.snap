// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Message About Room Builder Should build message 1`] = `
"<messageML>
  <card class=\\"barStyle\\" data-icon-src=\\"pagerduty-green-bg-icon.png\\">
    <header>
      <div style=\\"margin-bottom: 12px;\\"><b>Hello!</b></div>
      I'm the PagerDuty Bot. I'm available in your room <strong>All Technicians</strong> and I'll help you to keep updated about what happens on your services on PagerDuty . Here's how to start interacting with me following the commands below:
      <br />
      <br />
      <div>
        <div style=\\"padding-bottom: 4px;\\"><b style=\\"color: #767676\\">Commands</b></div>
        <div style=\\"border-left: 2px solid #767676;\\">
          <div style=\\"padding-left: 4px;\\">
            <strong>Help:</strong> used to help you with the commands available to interact with the PagerDuty Bot
            <br />
            <strong>Syntax:</strong> @Pagerduty bot <strong>help</strong>
          </div>
        </div>
      </div>
    </header>
  </card>
</messageML>"
`;
